src/content/cloud/volume-snapshots.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Volume Snapshots
+description: Use volume snapshots in your development environments
+sidebar_label: Volume Snapshots
+id: volume-snapshots
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Volume snapshots allows you to initialize persistent volume claims with the contents of a preexisting volume snapshot.
+Use volume snapshots when working with large datasets or if you want to create development environments with real data coming from your production or staging environments.
+
+## Requirements
+
+<Tabs
+  defaultValue="self-hosted"
+  values={[
+    { label: 'Self-Hosted', value: 'self-hosted', },
+    { label: 'SaaS', value: 'saas', },
+  ]}
+>
+<TabItem value="self-hosted">
+
+Follow [this guide](/docs/self-hosted/administration/volume-snapshots/) to enable Volume Snapshots in your cluster.
+
+</TabItem>
+
+<TabItem value="saas">
+If your instance is hosted by Okteto, Volume Snapshots are enabled by default.
+</TabItem>
+</Tabs>
+
+## Using Volume Snapshots in your Development Environment
+
+### Creating a Volume Snapshot
+
+Okteto allows developers to initialize persistent volume claims with the contents of any preexisting volume snapshot.
+The volume snapshot is created from a persistent volume claim and it can contain database backups, big files, images, a copy of your staging data, etc.
+
+The source persistent volume claim must have a storage class that supports volume snapshots. For Okteto SaaS users that means `csi-okteto-standard` or `csi-okteto-ssd`:
+
+<Tabs
+  defaultValue="kubernetes"
+  values={[
+    { label: 'Kubernetes', value: 'kubernetes', },
+    { label: 'Compose', value: 'compose', },
+  ]}
+>
+<TabItem value="kubernetes">
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mysql-pvc
+spec:
+  storageClassName: csi-okteto-standard
+  accessModes:
+  - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+```
+
+</TabItem>
+<TabItem value="compose">
+
+```yaml
+services:
+  mysql:
+    image: mysql
+    volumes:
+      - mysql-pvc:/var/lib/mysql
+
+volumes:
+  mysql-pvc:
+    driver_opts:
+      class: csi-okteto-standard
+```
+
+</TabItem>
+</Tabs>
+
+
+The following manifest creates a volume snapshot from the `mysql-pvc` persistent volume claim:
```

Comment:
Can you mention that this is the same process for both Kubernetes or docker compose?